modular_skyrat/code/modules/jobs/job_types/blueshield.dm

```diff
@@ -0,0 +1,54 @@
+/datum/job/blueshield
+	title = "Brig Physician"
+	flag = OFFICER
+	department_head = list("Captain")
+	department_flag = ENGSEC
+	faction = "Station"
+	total_positions = 1
+	spawn_positions = 1
+	supervisors = "the Captain"
+	selection_color = "#ddddff"
+	minimal_player_age = 7
+	exp_requirements = 200 //SKYRAT CHANGE - can't have inexperience players defending a head.
```

Comment:
This is in minutes. So currently you only need to play 3.33 hours. Maybe 2400 would be good.

Reply:
fuck bro.... ...